src/core/tests/type_prop/gather_elements.cpp

```diff
@@ -173,8 +173,8 @@ TEST(type_prop, gather_elements_out_of_bounds_axis) {
         auto GE = make_shared<op::v6::GatherElements>(D, I, axis);
         // Should have thrown, so fail if it didn't
         FAIL() << "axis out of bounds check failed";
-    } catch (const NodeValidationFailure& error) {
-        EXPECT_HAS_SUBSTRING(error.what(), std::string("axis must be within interval"));
+    } catch (const ov::AssertFailure& error) {
```

Comment:
nit: I would rather have node validation error -- it gives more information in the stderr